prime-router/src/main/kotlin/cli/tests/SftpcheckTest.kt

```diff
@@ -0,0 +1,120 @@
+package gov.cdc.prime.router.cli.tests
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import gov.cdc.prime.router.DeepOrganization
+import gov.cdc.prime.router.common.Environment
+import gov.cdc.prime.router.cli.SettingCommand
+import gov.cdc.prime.router.cli.SettingsUtilities
+import org.apache.http.HttpStatus
+import java.io.File
+
+/**
+ * Test SFTP receiver connections.  It checks the ignore.XYZ organization for the connection:
+ *  For each sftp receiver listed in the ignore organization, call the sftpcheck API
+ *  endpoint and verify you get back an OK response that we are able to connect to
+ *  the SFTP server.
+ *  This test needs to be able to run in staging and local, but not production.
+ *  When the smoke tests are running in sequential mode then have this new test
+ *  run after 'ping' and before 'end2end'.
+ *
+ * @returns:
+ *  If SUCCESS:
+ *      good("Test passed: SFTP")
+ *  If ERROR:
+ *      bad("Test SFTP receiver connections: Failed"
+ *
+ */
+
+class SftpcheckTest : CoolTest() {
+    override val name = "sftpcheck"
+    override val description = "Test SFTP receiver connections"
+    override val status = TestStatus.DRAFT
+
+    /**
+     * Define private local variables for use in the test.
+     */
+    private val receiverOrganizationPath = "./settings/organizations.yml"
```

Comment:
I like that we are using the local settings for this, but I wonder if we need to look at the settings for the specific environment.  This would mean connection to the settings API, get the list, etc., etc.  That makes it much more complicated thought.  @jimduff-usds ?